networkit/graphio.py

```diff
@@ -10,19 +10,17 @@
 from .GraphMLIO import GraphMLReader, GraphMLWriter
 from .GEXFIO import GEXFReader, GEXFWriter
 from . import algebraic
+from .support import MissingDependencyError
 
 # external imports
 import os
 import logging
+import numpy
+import scipy.io
 try:
-	import numpy
-except ImportError:
-	print("module 'numpy' not available - some functionality will be restricted")
-try:
-	import scipy.io
-except ImportError:
-	print("module 'scipy' not available - some functionality will be restricted")
-import fnmatch
+	import fnmatch
+except ImportError as importError:
+	fnmatch = None
```

Comment:
`fnmatch` is part of the Python standard library, so it must always be available.